facefusion/face_masker.py

```diff
@@ -9,23 +9,18 @@
 from facefusion.download import conditional_download_hashes, conditional_download_sources
 from facefusion.filesystem import resolve_relative_path
 from facefusion.thread_helper import conditional_thread_semaphore
-from facefusion.typing import FaceLandmark68, FaceMaskRegion, InferencePool, Mask, ModelOptions, ModelSet, Padding, VisionFrame
+from facefusion.typing import DownloadSet, FaceLandmark68, FaceMaskRegion, InferencePool, Mask, ModelSet, Padding, VisionFrame
 
 MODEL_SET : ModelSet =\
 {
-	'face_masker':
+	'face_occluder':
```

Comment:
this should remain face_masker as it includes face occluder and face parser

Reply:
how to handle size then?

Reply:
I resolved it already... you can merge